package.json

```diff
@@ -78,6 +78,7 @@
     "semantic-release": "^6.3.2",
     "sinon": "^2.1.0",
     "skin-deep": "^0.16.0",
+    "striptags": "^3.0.1",
```

Comment:
Because I'm getting this error without striptags in the root package.json:

    ERROR in JavaScript/Host.js from UglifyJs
    SyntaxError: Unexpected token: name (context) [./packages/neos-ui/~/striptags/src/striptags.js:17,0]